### PR TITLE
document frontmatter in help page

### PR DESCRIPTION
**Pull request by eddelbuettel**
In linl we added documentation for the frontmatter in the Rd which seems like a good idea.  So ditto here.

Minor cleanup to skeleton.Rd as well.

### CODE REVIEW

File: R/pinp.R [hunk 1]
```diff
@@ -11,6 +11,51 @@
 #' @return R Markdown output format to pass to
 #' \code{\link[rmarkdown:render]{render}}
 #'
+#' @section Document options:
+#' Various aspects of pdf document can be customized by setting either standard \pkg{rmarkdown}
+#' options, or any of the following options (which are shown in alphabetical order) in the 
+#' document metadata:
+#'
+#' \describe{
```
Comment (coatless) [inline R/pinp.R:19]: Have you tried `roxygen2` with Markdown enabled yet? 

https://cran.r-project.org/web/packages/roxygen2/vignettes/markdown.html#lists

Support exists for both `enumerate` and `itemize` lists. I manually recreate the `describe` list format by modifying the `itemize` list to mimic it using embedded segments. e.g.

```r
#' - `code`
#'     - description
```

**Review by coatless — APPROVED** [review]

File: R/pinp.R (hunk 1 above)
Comment (eddelbuettel) [reply R/pinp.R:19]: I happen to dislike what it requires in DESCRIPTION.

Comment (coatless) [reply R/pinp.R:19]: Aye, 'tis not a pretty sight; but, we can manually enable it on a per-chunk basis via:

```r
#' @md
```

Comment (coatless) [reply R/pinp.R:19]: With that being said, I'm fine with using the traditional latex markup.

**Pull request merged**